recipes/dpp/all/conanfile.py

```diff
@@ -0,0 +1,80 @@
+import os
+from conan import ConanFile
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMakeToolchain, CMake, cmake_layout, CMakeDeps
+from conan.tools.scm import Git
+from conan.tools.files import download, unzip
+
+required_conan_version = ">=1.65"
+
+class DPPConan(ConanFile):
+    name = "dpp"
+    license = "Apache-2.0"
+    package_type = "shared-library"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/brainboxdotcc/DPP"
+    description = "D++ is a lightweight and efficient library for Discord"
+    topics = ("discord")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False]}
+    default_options = {"shared": True}
```

Comment:
Note that if as per the comment above both static and shared are allowed, CCI recipes usually have shared: False as their default

Reply:
shared is better supported by our library than static, which is why we set the default to static: true. in fact, it is preferred that people don't build the library static unless they really know what they are doing as it will then bring in static openssl which is a security risk.

Reply:
I see. Then I'd suggest not allowing a static build in this case if that's ok with you (So removing the shared option from the recipe and keeping the package_type as-is)

Reply:
sounds like a plan, i'll do that